setup.py

```diff
@@ -59,40 +58,29 @@ def read(*parts):
 optional_libs = ['numexpr>=2.6.9']
 
 install_requires = [
-    'azure-storage-blob==12.3.2;python_version=="2.7"',
     'azure-storage-blob>=12.4.0;python_version>="3.7"',
```

Comment:
We no longer need all this `;python_version>="3.7"` everywhere, we can remove them.